runtime/vm/jfr.cpp

```diff
@@ -804,6 +823,91 @@ jfrExecutionSampleCallback(J9VMThread *currentThread, IDATA handlerKey, void *us
 	jfrExecutionSample(currentThread, currentThread);
 }
 
+void
+jfrCPULoad(J9VMThread *currentThread)
+{
+	PORT_ACCESS_FROM_VMC(currentThread);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+
+	omrthread_process_time_t currentProcCPUTime = {0};
+	intptr_t processTimeRC = omrthread_get_process_times(&currentProcCPUTime);
```

Comment:
This local should be named `currentProcCPUTimes` to better match the source and usage.